koa-crud/test/integration/controllers/vendor.spec.ts

```diff
@@ -0,0 +1,552 @@
+import chai, { expect } from 'chai';
+
+import chaiAsPromised from 'chai-as-promised';
+
+import mongoose from 'mongoose';
+
+import { Vendor } from '../../../src/lib/mongoose/models/vendor';
+
+import { initializeDatabase } from '../../../src/lib/mongoose';
+
+import {
+  selectAllVendorsController,
+  selectOneVendorController,
+  insertVendorController,
+  updateVendorController,
+  deleteOneVendorController,
+} from '../../../src/controllers/vendors';
+
+chai.use(chaiAsPromised);
+
+const mockedId = mongoose.Types.ObjectId().toString();
+
+describe('Vendor Controller', () => {
+  before(() => initializeDatabase());
```

Comment:
You can add `mockedId` in this `before` hook.

Reply:
noted sir